app/assets/javascripts/democracy.js

```diff
@@ -365,6 +365,23 @@ function start_end_fdatetimepicker(start, end, min_minutes, suggested_minutes) {
         });
 }
 
+function fdatetimepicker_only_date(start, end) {

```

Comment:
it's ok here for now. I introduced an airesis.coffee where I would like to move all our global functions.